config.py

```diff
@@ -6,7 +6,7 @@ class Config(object):
 class DevelopmentConfig(Config):
     """Configurations for Development."""
     DEBUG = True
-    SECRET_KEY = "samplesecret"
+    SECRET_KEY = os.getenv('SECRET_KEY')
```

Comment:
Good implementation, although doesn't it take a second argument of the secret key?